lib/cbus_elixir_web/controllers/page_controller.ex

```diff
@@ -1,11 +1,26 @@
 defmodule CbusElixirWeb.PageController do
+  import Ecto.Query, only: [from: 2]
   use CbusElixirWeb, :controller
 
   alias CbusElixir.Repo
   alias CbusElixir.App.Speaker
+  alias CbusElixir.App.Meeting
 
   def index(conn, _params) do
-    speakers = Repo.all(Speaker)
-    render conn, "index.html", speakers: speakers
+    today = Timex.today()
+    meeting = Repo.all(from m in Meeting, 
```

Comment:
This seems similar to calling `ActiveRecord::Base.connection.select_all` back in Rails - a low-level interface that skips most of the mapping layer.

I recommend [this blog post](http://blog.plataformatec.com.br/2015/08/working-with-ecto-associations-and-embeds/) for more about Ecto associations. We should use them. :)